pkg/controller/replication/replication_controller.go

```diff
@@ -112,7 +110,7 @@ func (r *ReconcileReplication) Reconcile(request reconcile.Request) (reconcile.R
 		}
 		result = reconcile.Result{Requeue: true}
```

Comment:
same here

Reply:
there are some more occurrences, maybe we should drop the named return variables entirely? I always get confused about the origin of the returned value. Also, in this case, most of the assigned return values are never returned.